utils/node_local_test.py

```diff
@@ -587,7 +605,7 @@ def start(self):
         if self.conf.args.no_root:
             cmd.append('--recreate-superblocks')
 
-        self._sp = subprocess.Popen(cmd)
+        self._sp = subprocess.Popen(cmd, env=plain_env)
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations